    lista = [1, 2, 3, 4, 5]

    try:

    lista[10]

    except IndexError:

    print('Error: El dato esta fuera de rango')

